src/components/pages/Chat/RoomDropdown/index.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useEffect, useRef } from 'react'
+import { useState } from 'react'
+import styled from 'styled-components'
+import {
+    AddMemberIcon,
+    CopyIcon,
+    DeleteIcon,
+    DownloadIcon,
+    EditMembersIcon,
+    SearchIcon,
+} from '../../../../icons'
+import { KARELIA } from '../../../../utils/css'
+
+type Props = {
+    button?: React.ReactElement
+}
+
+const DropDownContainer = styled.div`
+    margin-left: 10px;
+`
+const DropDownHeader = styled.div`
+    margin-bottom: 0.8em;
+    padding: 0 2em 0 1em;
+    box-shadow: 0 2px 3px rgba(0, 0, 0, 0.15);
+    font-weight: 500;
+    font-size: 1.3rem;
+    color: #3faffa;
+    background: #ffffff;
+`
+const DropDownListContainer = styled.div`
+    position: absolute;
+    top: 75px;
+    right: 35px;
+    z-index: 100;
+`
+const DropDownList = styled.div`
+    width: 250px;
+    padding: 0;
+    margin: 0;
+    background: #ffffff;
+    border-radius: 10px;
+    box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.1);
+    color: #59799c;
+    font-family: ${KARELIA};
+    font-size: 14px;
+    list-style-type: none;
+    border-spacing: 0px;
+
+    hr {
+        border: 0;
+        border-top: 1px solid #dee6ee;
+        padding: 0px;
+        margin: 0px;
+    }
+`
+const ListItem = styled.div`
```

Comment:
Let's make this a `styled.button`, for semantics. This or a link but I think a button is more suitable.

Remember to add `appearance: none` property.